src/app/feed/feed-post-icon-row/feed-post-icon-row.component.html

```diff
@@ -1,152 +1,350 @@
-<div class="js-feed-post-icon-row__container fs-14px text-grey5 d-flex justify-content-between unselectable">
+<section>
   <div
-    class="cursor-pointer d-flex align-items-center"
-    (click)="openModal($event)"
-    data-toggle="modal"
-    data-target=".js-feed-post-icon-row__comment-modal"
+    *ngIf="postReactionCounts.Total > 0 && !hideSummary"
```

Comment:
`hideSummary` is a prop that controls if we show reactions summary or not (when you are on a Post page the summary is hidden, and you have only counter like `19 Reactions`

<img width="663" alt="Screenshot 2023-02-03 at 13 30 00" src="https://user-images.githubusercontent.com/10476834/216592733-21d2db04-aa00-4d66-aef3-d5201423bdaa.png">